src/libsyntax/ast.rs

```diff
@@ -821,7 +821,7 @@ impl Stmt {
 
     pub fn is_item(&self) -> bool {
         match self.node {
-            StmtKind::Local(_) => true,
+            StmtKind::Item(_) => true,
```

Comment:
:laughing:

Reply:
Hey, you missed it too. I don't wanna hear it 😛